tests/test_integration.py

```diff
@@ -245,6 +271,42 @@ def test_start_job_with_input_samplesheet(self):
             "socks_samplesheet_samplesheet.csv"
             in " ".join(status_response_body["command"])
         )
+        self.assertTrue(
+            "--demultiplexer" in status_response_body["command"]
```

Comment:
This will result in adding `--demultiplexer` with no value, right? If the purpose is to test what we can add a flag parameter (that is, a parameter that is true if it is used), I would suggest calling the variable `test_flag_pipeline_param` or similar.

Reply:
To be clear, I don't think we need support for flag parameters as of now, I just wanted to understand what you were testing. :)  If I remember correctly the request body contained `{"demultiplexer": ""}`. I think it has been removed now though.